PowerPointLabs/PowerPointLabs/ActionFramework/TooltipsLab/Create/CreateTrigger/CreateTriggerActionHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Windows;
+using PowerPointLabs.ActionFramework.Common.Attribute;
+using PowerPointLabs.ActionFramework.Common.Extension;
+using PowerPointLabs.ActionFramework.Common.Interface;
+using PowerPointLabs.Models;
+using PowerPointLabs.TextCollection;
+using PowerPointLabs.TooltipsLab;
+using PowerPoint = Microsoft.Office.Interop.PowerPoint;
+
+namespace PowerPointLabs.ActionFramework.TooltipsLab
+{
+    [ExportActionRibbonId(TooltipsLabText.CreateTriggerTag)]
+    class CreateTriggerActionHandler : ActionHandler
+    {
+        protected override void ExecuteAction(string ribbonId)
+        {
+            this.StartNewUndoEntry();
+
+            PowerPointSlide currentSlide = this.GetCurrentSlide();
+
+            PowerPoint.Selection selection = this.GetCurrentSelection();
+
+            if (selection.Type != PowerPoint.PpSelectionType.ppSelectionShapes)
+            {
+                MessageBox.Show("Please select 1 or more shapes as your callout shape.");
+                return;
+            }
+
+            foreach (PowerPoint.Shape selectedShape in selection.ShapeRange)
+            {
+                // TODO: Adding a trigger to an existing shape that is already part of a trigger animation as the "callout" (whether created by user or by TooltipsLab)
```

Comment:
This will be addressed in a future issue? :)

Reply:
Turns out that this was intended behaviour. I added this todo wrongly, haha